### ISSUE TITLE
Shadow of War compatibility

### ISSUE THREAD

**Issue by ketacz** [x360ce/x360ce#687]
Hello!

I'm using your software for a longer bit now and it used to work on a lot of games - cheers for that!

Sadly, I'm unable to make it work on Shadow of War game which is a pain to play with k&m.

Are there any plans to make it work here?

Appreciate it

edit.

Ok I just found that the game won't work since it's Denuvo game or whatever....

Sadpanda then :(

**Comment by Kygies27** [comment]
I managed to get it to work, but the game has to be cracked, I renamed XInput file to 'XInput9_1_0.dll', and downloaded the 'DInputx64.dll' file from the X360ce and added it to the game folder (X64).

It seems to work just fine for the first few hours. Note that cracking your game is a legal gray area and may (and by 'may' I mean almost certain) violate your EULA with WB. However, as a consumer, you probably don't care. Just letting you know anyways.
